releng/pom.xml

```diff
@@ -28,7 +28,7 @@
 		</repository>
 		<repository>
 			<id>maven</id>
-			<url>${JENKINS_URL}/job/lsp4j/job/master/lastStableBuild/artifact/build/maven-repository</url>
+			<url>http://services.typefox.io/open-source/jenkins/job/lsp4j/job/master/lastStableBuild/artifact/build/maven-repository</url>
```

Comment:
we dont have lsp4j on xtext jipp

Reply:
correct

Reply:
maybe put this into a separate change, since LSP update not yet mergable

Reply:
no this change builds against snapshots (version change got lost in my previous commit)

Reply:
I mean the url change. This should go to master, it is simply wrong.

Reply:
https://github.com/eclipse/xtext-core/pull/954 is meant to be on master now. the release change is at  https://github.com/eclipse/xtext-core/pull/955

Reply:
#954 is this one?

Reply:
yes thus 'lsp4j': '0.6.0-SNAPSHOT',

Reply:
=> this pr builds against snapshots. the other will set it on release at release say (30.11)

Reply:
@kthoms ok to merge?